src/pages/zh-CN/comparing-astro-vs-other-tools.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Astro vs. X
+description: 比较 Astro 和诸如 Gatsby、Next.js、Nuxt、Hugo、Eleventy 之类的静态网站生成器。
+---
+<!-- TODO: UNcomment out the parts re: number of bytes of JS etc, once we decide which values/markers we'd like to use here. -->
+我们经常会被问道这样一个问题“Astro 和我最喜欢的 **\_\_\_\_** 项目比起来怎么样？”
+
+本指南就是为了回答这个问题而编写的，适用于几个流行的网站生成器和 Astro 替代品。
+
+两个重要特性使得 Astro 与大多数替代品不同。
+
+- [按需加载](/en/core-concepts/partial-hydration)
+- [使用你喜欢的框架](/en/core-concepts/framework-components)
+
+你可以在这个页面上通过我们的深入比较了解到更多细节。
+
+如果你最喜欢的生成器没有列在下方，你可以 [在 Discord 里问我们](https://astro.build/chat)。
+
+## Docusaurus vs. Astro
+
+[Docusaurus](https://docusaurus.io/) 是一个流行的文档网站生成器。Docusaurus 使用 React 来生成网站 UI，而 Astro 支持 React、Preact、Vue.js、Svelte、SolidJS、AlpineJS、Lit 和原始 HTML 模板。
+
+Docusaurus 是为建立文档网站而设计的，它有一些内置的、针对文档的网站功能，而 Astro 没有。不够 Astro 通过官方的 [`docs`](https://github.com/withastro/astro/tree/main/examples/docs) 主题提供了特定的文档功能，你可以在自己的网站上使用。这个网站正是用这个模板建立的！
+
+#### 比较 Docusaurus vs. Astro 性能
+
+在大多数情况下，Astro 网站的加载速度将明显快于 Docusaurus 网站。这是因为 Astro 会自动从页面中剥离不必要的 JavaScript，只激活个别需要的组件。这个功能被称为[部分激活](/en/core-concepts/partial-hydration)。
+
+Docusaurus 不支持部分激活，而是让用户在浏览器中加载并重新激活整个页面，即便大部分的页面内容都是静态的，这也使得网站页面加载比较慢和性能比较差。在 Docusaurus 中没有办法禁用这种行为。
+
+#### 案例：创建文档站点
+
+[docusaurus.io/docs](https://docusaurus.io/docs) 是 Docusaurus 官方文档网站，不言而喻它自然是用 Docusaurus 构建的。与 Astro 官方文档网站相比较，二者的设计和功能十分相似。下面给出了两个站点生成器的**粗略但又真实**的比较。
+
+- **Docusaurus 性能得分**：53 分（满分 100 分）[（全面检查）](/lighthouse/docusaurus/)
+- **Astro 性能得分**：92 分（满分 100 分）[（全面检查）](/lighthouse/astro/)
+
+<!-- One big reason behind this performance difference is Astro’s smaller JavaScript payload: [docusaurus.io/docs](https://docusaurus.io/docs) loads **238kb** of JavaScript on first page load while [docs.astro.build](https://docs.astro.build) loads **78.7kb** (67% less JavaScript, overall) _after_ first load.) -->
+
+## Elder.js vs. Astro
+
+[Elder.js](https://elderguide.com/tech/elderjs/) 是一个有态度、为 Svelte 打造的静态网站生成器。
+
+Elder.js 使用 Svelte 来渲染网站内容。Astro 更灵活：你可以自由地使用任何流行的组件库（React、Preact、Vue、Svelte、Solid 和其他）或 Astro 的类似 HTML 的组件语法来构建 UI，这类似于 HTML + JSX。
+
+Elder.js 在这个列表中是独一无二的，因为它是唯一个支持[部分激活](/en/core-concepts/partial-hydration)的生成器。Astro 和 Elder.js 都会自动从页面中剥离不必要的 JavaScript，只激活个别需要的组件。Astro 与 Elder 的部分激活在 API 方面有点不同，Astro 支持一些 Elder.js 没有的功能（如 `client:media`)。然而就性能而言，这两个项目构建的网站基本一致。
+
+Elder.js 使用自定义的路由解决方案，对于新的开发者来说可能会感到陌生。而 Astro 则使用[基于文件的路由](/en/core-concepts/routing)，这对任何来自 Next.js\SvelteKit，甚至其他静态网站生成器如 Eleventy 的人来说都应该感到熟悉。
```

Comment:
```suggestion
Elder.js 使用自定义的路由解决方案，对于新的开发者来说可能会感到陌生。而 Astro 则使用[基于文件的路由](/zh-CN/core-concepts/routing)，这对任何来自 Next.js\SvelteKit，甚至其他静态网站生成器如 Eleventy 的人来说都应该感到熟悉。
```

Reply:
This page has not been translated, so it should not be changed.

I plan to do a full revision when all pages are translated.

Reply:
We should use `/zh-CN/` links here! That way we don’t need to go back and update them later — the page already exists: https://docs.astro.build/zh-CN/core-concepts/routing/

Reply:
Have I translated it yet? I didn't pay attention to it.

But I still prefer to update it with the last, otherwise every time I submit a translation I still need to check if there is a link to this page in the previous translation, and then modify it.

Reply:
It's will make ci error, right? Should we ignore it.

Reply:
No, it will pass. The CI checks the actual built URLs rather than which `.md` files exist.